drivers/adc/ad9250/ad9250.h

```diff
@@ -513,6 +513,8 @@ struct ad9250_dev {
 struct ad9250_init_param {
 	/* SPI */
 	spi_init_param	spi_init;
+	/* Device Settings */
+	struct ad9250_state ad9250_st_init;
```

Comment:
i would just name the structure member `state` or something generic. `ad9250_` is redundant since it is part of an `ad9250_init_param` struct.

Reply:
I agree. I just wanted keep consistency within the driver.
https://github.com/analogdevicesinc/no-OS/blob/7d37ec72a03168914854d309816894b984580938/drivers/adc/ad9250/ad9250.h#L509

I think renaming all structure parameters should be part of another commit.

Reply:
that could do it then